flow-model-generator/modules/flow-model-generator-ls-extension/src/test/resources/node_template/config/new_xml_payload.json

```
@@ -0,0 +1,49 @@
+{
+  "description": "Test generating template for new xml payload node",
+  "codedata": {
+    "node": "NEW_XML_PAYLOAD_DATA"
+  },
+  "output": {
+    "id": "31",
+    "metadata": {
+      "label": "New XML Payload"
+    },
+    "codedata": {
+      "node": "NEW_XML_PAYLOAD_DATA"
+    },
+    "returning": false,
+    "properties": {
+      "variable": {
+        "metadata": {
+          "label": "Data variable",
+          "description": "Name of the variable"
+        },
+        "valueType": "IDENTIFIER",
+        "value": "item",
+        "optional": false,
+        "editable": true
+      },
+      "type": {
+        "metadata": {
+          "label": "Data type",
+          "description": "Type of the variable"
+        },
+        "valueType": "TYPE",
+        "value": "xml",
+        "optional": false,
+        "editable": true
+      },
+      "expression": {
+        "metadata": {
+          "label": "Expression",
+          "description": "Create new XML payload"
+        },
+        "valueType": "EXPRESSION",
+        "value": "xml``",
```

IMO, we shouldn't display the XML template expression syntax to the user as it is Ballerina-specific and not intended for an integration developer. Therefore, only the XML within the backtick string should be sufficient.

```suggestion
        "value": "",
```